include/threadClass.hpp

```diff
@@ -1,3 +1,28 @@
+/**
+ *  Schulich Delta OnBoard Media Control
+ *  Copyright (C) 2015 University of Calgary Solar Car Team
+ *
+ *  This file is part of the Schulich Delta OnBoard Media Control
+ *
+ *  The Schulich Delta OnBoard Media Control is free software: 
+ *  you can redistribute it and/or modify it under the terms 
+ *  of the GNU Affero General Public License as published by 
+ *  the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  The Schulich Delta OnBoard Media Control is distributed 
+ *  in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+ *  without even the implied warranty of MERCHANTABILITY or 
+ *  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero 
+ *  General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General 
+ *  Public License along with the Schulich Delta OnBoard Media 
+ *  Control.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  For further contact, email <software@calgarysolarcar.ca>
+ */
```

Comment:
So Enoch. Only reason I have not accepted pull request is cause I am awaiting reply for what we should do on a file that isn't written by us. In this file it has its credit going to:
`/** CREDIT: http://stackoverflow.com/a/1151615 **/` 

The stack overflow response had a complete class where it handled creating wrapper class that allowed threading. Did you want us to re-write this or is there something special we do?